src/main/scala/ztis/KafkaComponent.scala

```diff
@@ -7,15 +7,13 @@ import com.twitter.chill.{KryoInstantiator, KryoPool}
 import com.typesafe.config.Config
 import com.typesafe.scalalogging.slf4j.StrictLogging
 import kafka.consumer.ConsumerConfig
-import ztis.twitter.TweetKryoRegistrar
-import ztis.wykop.WykopKryoRegistrar
 
 import scala.collection.JavaConverters._
 
 
 trait KafkaComponent extends StrictLogging {
-  val kryo = KryoPool.withByteArrayOutputStream(1, (new KryoInstantiator)
-    .withRegistrar(new TweetKryoRegistrar).withRegistrar(new WykopKryoRegistrar))
+  private val kryoInstantiator = (new KryoInstantiator).withRegistrar(new ZTISKryoRegistrar)
+  val kryo = KryoPool.withByteArrayOutputStream(1, kryoInstantiator)
```

Comment:
KryoPool is a class from twitter-chill library: https://github.com/twitter/chill and this class is written in Java so there is no access to named arguments. I will extract the 1 value to private val and give it some meaningful name.